triton/tut/interactive.rst

```diff
@@ -2,160 +2,190 @@
-session running, just closing the terminal isn't enough. Exit all
-shells in the screen session on the node (C-d or ``exit``), or cancel
-the process (see below).**
+.. warning::
 
-If you are off-campus, you might want to use https://vdi.aalto.fi as a
-virtual desktop to connect to Triton to run graphical programs.
-Otherwise, expect things to be very slow.
+  Just like with ``srun --pty bash``, remember to exit the shell.
+  Since there is a separate screen session running, just closing the terminal isn't enough. 
+  Exit all shells in the screen session on the node (C-d or ``exit``), or cancel
+  the process (see below).
 
+.. note::
 
-More options
-============
+  If you are off-campus, you might want to use https://vdi.aalto.fi as a
+  virtual desktop to connect to Triton to run graphical programs.
+  Otherwise, expect things to be very slow.
+
+Monitoring your usage
+=====================
 
-**Time/CPU/memory requirements:** The commands srun/sinteractive have
-many more options that let you specific resources. The most important
-for interactive running are probably ``--mem``, ``--cpus-per-task`` (``-c``),
-and ``--time`` (``-t``).
-
-**How much time/memory/CPU resources should you request?**  The less
-you request, the faster you are likely to run. As for all you need, but
-not ridiculously large amounts. If you request something slightly less
-than a node size (note that we have different size nodes) or partition
-limit (see below), you are more likely to fit into a spare spot. We
-have many nodes with 12 cores, and some with 20 or 24. If you request
-24, you have very limited options. If you request 10, or 18, you will
-have a lot more options. Same with memory: most common cutoffs are 48,
-64, 128, 256GB. Use smaller values when interactive testing, then more
-for batch running overnight.
-
-**Configure your program well:** Also, note that requesting more CPUs
-doesn't magically mean that your program becomes parallel. Make sure
-you turn that on in your code to enable that. Also specify how many
-CPUs to use (matching how many you request with slurm). If you don't
-get an entire node, your program might try to use all CPUs, and the OS
-will limit the number you can use (with cgroups, if you are
-interested). This leads to inefficiency.
-
-**Partitions:** Now almost always automatically set, but used to be
-important.  Partitions are groups of nodes reserved for different
-purposes.  For example, ``-p interactive`` tells us to us the
-interactive partition - which should always be available for quick
-tests.  ``-p debug`` is a short partition for debugging.  See a bit
-more in the :doc:`serial tutorial <serial>`.
+When your jobs enter the queue, you need to be able to get
+information on how much time, memory, etc. your jobs are using 
+in order to know what requirements to ask for. 
 
+The command ``slurm h`` gives you information such as the actual memory used by your recent jobs, total CPU time, etc.
```

Comment:
```suggestion
The command ``slurm history`` gives you information such as the actual memory used by your recent jobs, total CPU time, etc.
```
Full command to make it easier to remember?  Which do you think is easier?

Reply:
Ah yes. I could write the full command and then also introduce the short one.